app/src/main/java/protect/card_locker/Utils.java

```diff
@@ -24,6 +24,8 @@
 import android.widget.TextView;
 import android.widget.Toast;
 
+import androidx.annotation.IntRange;
```

Comment:
unused import

(will review the rest properly later)

Reply:
Are you sure you're looking at the right commit? Because there's *definitely* no `import androidx.annotation.IntRange;` on my end

Reply:
You removed it again in a later commit. Like I said, I haven't reviewed properly yet so I missed that :)

Reply:
I was looking at the commit separately because you mentioned me, thus not seeing that was outdated in the context of the PR.